src/Feedback.Analyzer.Backend/Feedback.Analyzer.Api/Configurations/HostConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Feedback.Analyzer.Api.Configurations;
+
+public static partial class HostConfiguration
+{
+    /// <summary>
+    /// Configures application builder
+    /// </summary>
+    /// <param name="builder">Application builder</param>
+    /// <returns>Application builder</returns>
+    public static ValueTask<WebApplicationBuilder> ConfigureAsync(this WebApplicationBuilder builder)
+    {
+        builder
+            .AddExposers()
+            .AddDevTools();
+        return new(builder);
+    }
+    
+    /// <summary>
+    /// Configures application
+    /// </summary>
+    /// <param name="app">Application host</param>
+    /// <returns>Application host</returns>
+    public static  ValueTask<WebApplication> ConfigureAsync(this WebApplication app)
```

Comment:
Add  `async`  keyword to public static ValueTask<WebApplication> ConfigureAsync()

Reply:
here async keyword is written later, for example, when registering seed data

Reply:
Ok